fix: validate PGBOUNCER_URLS on config gen (#174)
heroku/heroku-buildpack-pgbouncer: README.md

    HEROKU_POSTGRESQL_ROSE_URL=postgres://u9dih9htu2t3ll:password@127.0.0.1:6000/db2
    DATABASE_URL=postgres://uf2782hv7b3uqe:password@127.0.0.1:6000/db1

> > ⚠️ A referenced configuration variable in `PGBOUNCER_URLS` must not be empty, or otherwise must be a valid PostgreSQL connection string.

Reviewer: ```suggestion
> ⚠️ A referenced configuration variable in `PGBOUNCER_URLS` must not be empty, and must be a valid PostgreSQL connection string.
```
